cogs/ahk/ahk.py

```diff
@@ -304,9 +304,11 @@ async def cloudahk_call(self, ctx, code, lang='ahk'):
 			' No output.\n' if stdout == '' else '\n```autoit\n{0}\n```'.format(stdout),
 			'Timed out' if time is None else '{0:.1f} seconds'.format(time),
 		)
-
-		await ctx.send(out)
-
+		try:
+			await ctx.send(content=out,reference=ctx.message.reference)
```

Comment:
This change doesn't make much sense in my opinion. I think it is confusing to have a response that mentions one user (the one who invoked the command) in the body, while also including a reply to a different user (if the invoker was replying to someone else).

For example, reading this is a bit of a mess, and hard to follow:

![image](https://user-images.githubusercontent.com/43710474/106319485-20ec1c00-6237-11eb-8f06-00694d426f01.png)

I think that replacing the mention with a reply to the invoker would be a better option.

Reply:
That's what should've happened. 
```suggestion
			await ctx.send(content=out, reference=ctx.message.to_reference())
```
That's what the line of code should be.